pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -48,6 +48,16 @@ const (
 
 	// defaultConfig is the default service config (that does not correspond to an actual service)
 	defaultConfig = "default"
+
+	// policyCheck is the node label holding the policy check option override
+	policyCheck = "POLICY_CHECK"
```

Comment:
That works - but I'm not sure it's the simplest solution. 

We do pass all annotations - there is no need to add an extra redundant metadata. 
@mandarjog  to confirm that all annotations are already passed.

Reply:
Yes, all annotations are available to pilot.

Reply:
Why is there META_INTERCEPTION_MODE? How do I access pod annotations in pilot?

Reply:
They are available in the `*plugin.InputParams.Node.Metadata` object.